uber/site_sections/registration.py

```diff
@@ -101,11 +101,16 @@ def form(self, session, message='', return_to='', omit_badge='', check_in='', **
                     attendee.checked_in = localized_now()
                 session.add(attendee)
                 if return_to:
-                    raise HTTPRedirect(return_to + '&message={}', 'Attendee data uploaded')
+                    raise HTTPRedirect(return_to + '&message={}', 'Attendee data saved')
                 else:
-                    raise HTTPRedirect('index?uploaded_id={}&message={}&search_text={}', attendee.id,
-                        '{} has been uploaded'.format(attendee.full_name),
-                        '{} {}'.format(attendee.first_name, attendee.last_name) if c.AT_THE_CON else '')
+                    msg_text = '{} has been saved'.format(attendee.full_name)
+                    if c.AT_THE_CON:
+                        # redirect back to search page for registration staffers to quickly process the next checkin
+                        raise HTTPRedirect('index?uploaded_id={}&message={}&search_text={}', attendee.id, msg_text,
+                            '{} {}'.format(attendee.first_name, attendee.last_name) if c.AT_THE_CON else '')
```

Comment:
Since we're already inside the ``if c.AT_THE_CON`` we don't need a further ``if c.AT_THE_CON`` expression here and can instead just list the first and last name.